layouts/partials/head.html

```diff
@@ -43,6 +43,8 @@
 <link rel="stylesheet" href="{{ "css/prism.css" | relURL }}"/>
 {{ end -}}
 
+{{ partial "css.html" . -}}
```

Comment:
Please put this inside `layouts/partials/hooks/head-end.html`

It belongs there (and I missed it in a previous rebase of https://github.com/kubernetes/website/pull/48350)